backend/app/api/jobs.py

```
@@ -0,0 +1,121 @@
+        # Save to MongoDB
+        await job_application.insert()
+        
+        return JobUrlResponse(
+            success=True,
+            message="Job URL submitted successfully",
+            request_id=job_request.request_id,
+            user_id=job_request.user_id,
+            timestamp=job_request.timestamp
+        )
+        
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)